demo/c/README.md

```diff
@@ -9,7 +9,16 @@ You need a C99-compatible compiler to build these demos.
 - The microphone based demo requires [miniaudio](https://github.com/mackron/miniaudio) for accessing microphone audio data. 
 - The following demo must be run at the [root](/../../) of the directory.
 - **For Windows Only**: [MinGW](http://mingw-w64.org/doku.php) is required to build the demo.
-   
```

Comment:
The main readme is not updated, is there any reason for that?

Reply:
Me forgetting it I suppose :)